src/Components/PresentationalCard/DescriptionCard.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {
+  Card,
+  CardBody,
+  Grid,
+  GridItem,
+  TextContent,
+  Text,
+  TextVariants,
+  Flex,
+  Button,
+} from '@patternfly/react-core';
+
+const DescriptionCard = () => {
+  return (
+    <>
```

Comment:
Extra return statement, and blank element. I do prefer `React.Fragment` as they are more visible. although it's not required here at all. There's just one card as child element.
```suggestion
const DescriptionCard = () => (
  <Card className="pf-u-mb-xl">
```